integration-test/src/main/resources/testsuites/e2e/aws-e2e-tests.yaml

```diff
@@ -20,4 +20,5 @@ tests:
       - name: com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.SdxImagesTests
       - name: com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.SdxSecurityTests
       - name: com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.SdxRecipeTests
-      - name: com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.SdxRangerRazEnabledTests
\ No newline at end of file
+      - name: com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.SdxRangerRazEnabledTests
+      - name: com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.SdxResizeTests
```

Comment:
This is a duplicated test reference for the `com.sequenceiq.it.cloudbreak.testcase.e2e.sdx.SdxResizeTests`. If the test execution time is more than 150 minutes, please remove this line from here.

Reply:
addressed.